packages/plugin/tests/processor-without-config.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { processor } from '../src/processor';
+
+jest.mock('../src/graphql-config');
```

Comment:
here we no need to mock `graphql-config` and load the default `virtual` graphql-config